File: content/fixtures/code.py
from __future__ import annotations

from dataclasses import dataclass


@dataclass(frozen=True)
class ReviewNote:
    section: str
    owner: str
    message: str


def build_release_notes(notes: list[ReviewNote]) -> list[str]:
    return [
        f"{note.section}: {note.owner} confirmed {note.message}"
        for note in notes
        if note.owner
    ]


def main() -> None:
    notes = [
        ReviewNote("inventory", "Mina", "cycle counts are balanced"),
        ReviewNote("quality", "Owen", "extraction fixture marker: shared code sample"),
    ]

    for line in build_release_notes(notes):
        print(line)


if __name__ == "__main__":
    main()

File: content/fixtures/code.cpp
#include <iostream>
#include <string>
#include <vector>

struct ReviewItem {
    std::string owner;
    std::string status;
    std::string note;
};

int main() {
    const std::vector<ReviewItem> items = {
        {"Ada", "done", "header checks complete"},
        {"Lin", "waiting", "extraction fixture marker: shared code sample"},
        {"Sam", "done", "footer totals verified"},
    };

    for (const auto& item : items) {
        std::cout << item.owner << " - " << item.status << ": " << item.note << '\n';
    }

    return 0;
}
